Assignment submission

asamartzi opened the pull request:
This is the final submission for the Junior Web Developer Assignment by Anastasia Samartzi.

asamartzi committed:
initial_commit

Web assignment

asamartzi committed:
Update README.md

asamartzi committed:
Update README.md

minor changes

akalipetis commented on web_assignment/app.py:
```diff
@@ -0,0 +1,50 @@
+import json
+
+app = Flask(__name__)
+app.secret_key = "mysecretkey"
```
Better get the secret key for the environment ([`os.getenv`](https://docs.python.org/3/library/os.html#os.getenv)) and fallback to a static value if it does not exist.

akalipetis commented on web_assignment/app.py:
```diff
@@ -0,0 +1,50 @@
+        print(final_lst)
+        print("Your list is : ")
+        for l in final_lst:
+            print(l)
```
This is great for debugging, but it's not needed here 😄

akalipetis commented on web_assignment/app.py:
```diff
@@ -0,0 +1,50 @@
+        #Turn the sorted list in a JSON list and return it. 
+        json_list = json.dumps(final_lst)
+
+        return json_list
```
Better use [`jsonify`](https://flask.palletsprojects.com/en/1.1.x/api/#flask.json.jsonify) to return the response.

akalipetis reviewed (changes requested):
Thanks a lot for your submission @asamartzi!

I've added a first round of comments, please fix those in order for us to do a final review.

Also, please make sure you remove all `*.pyc` and `.DS_Store` files and add them in `.gitingore`.

asamartzi committed:
comment_fixes

Fixed the 3 comments submitted and updated .gitignore file as well.

asamartzi committed:
Delete .DS_Store

asamartzi committed:
Delete .DS_Store

asamartzi committed:
Delete web_assignment/__pycache__ directory

asamartzi committed:
Delete .DS_Store

asamartzi commented:
@akalipetis 
Thank you for the feedback.
I changed the parts you commented and also added the .gitignore as suggested.
Let me know for any additional suggestions.
